### PR TITLE
Enable nvfuser tests for refs.broadcast_to and refs.broadcast_tensors

### PR DESCRIPTION
**Pull request by IvanYashchuk**
Previously these tests were failing because they required some other op alongside prims.broadcast_in_dim to be executed. Now it works standalone.

### CODE REVIEW

File: test/test_proxy_tensor.py [hunk 1]
```diff
@@ -1069,7 +1069,6 @@ def f(a, b):
     xfail('mode', ''),  # aten.mode.default - couldn't find symbolic meta function/decomposition
     xfail('msort', ''),  # aten.sort.default - couldn't find symbolic meta function/decomposition
     xfail('mv', ''),  # aten.mv.default - couldn't find symbolic meta function/decomposition
-    xfail('nanmean', ''),  # The underlying op of 'aten.stride' has no overload name '_schema'
```
Comment (IvanYashchuk) [inline test/test_proxy_tensor.py:1072]: Not sure what happened here.

**Review by mruberry — APPROVED** [review]
Cool! Stamped!

**Review by ngimel — APPROVED** [review]

**Pull request closed without merging**